src/components/parser.cpp

```diff
@@ -133,7 +133,7 @@ void parser::codeblock(string&& data, const bar_settings& bar) {
         break;
 
       case 'O':
-        m_sig.emit(offset_pixel{static_cast<int>(std::strtol(value.c_str(), nullptr, 10))});
+        m_sig.emit(offset{string{value}});
```

Comment:
I think the parser should parse the value of the offset tag and only send the renderer either a direct pixel value or a `struct geometry`. The latter probably makes more sense since the parser shouldn't have to bother with dpi.

Reply:
Fixed by https://github.com/jaagr/polybar/pull/1671/commits/f2c170c6c69522461f808ba0a762430366dc192e